xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -602,9 +602,11 @@ void CGUIWindowPictures::OnItemInfo(int itemNumber)
 
 std::string CGUIWindowPictures::GetStartFolder(const std::string &dir)
 {
-  std::string lower(dir); StringUtils::ToLower(lower);
-  if (lower == "plugins" || lower == "addons")
+  if (StringUtils::EqualsNoCase(dir, "plugins") ||
```

Comment:
you've changed the implementation from equals to equals no case and not only the readability ;)